var/spack/repos/builtin/packages/pandoramonitoring/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Pandoramonitoring(CMakePackage):
+    """Metadata package to bring together and build multiple Pandora libraries.
+    NOTE: for proper version control with spack, this should be broken up and
+    the subpackages installed individually."""
+
+    url = "https://github.com/PandoraPFA/PandoraMonitoring/archive/v03-04-00.tar.gz"
+    homepage = "https://github.com/PandoraPFA/PandoraMonitoring"
+    git = "https://github.com/PandoraPFA/PandoraMonitoring.git"
+
+    tags = ["hep"]
+
+    maintainers("vvolkl", "wdconinc")
```

Comment:
```suggestion
    maintainers("jmcarcell", "wdconinc")
```
If you want you can add me instead of adding Valentin to more packages since he's working on other things now